docker/tds/docker-compose.yml

```diff
@@ -20,25 +35,26 @@ services:
     ports:
       - "32843:8080"
     env_file: tds-docker.env
-  exam_mysql:
-    image: mysql
-    ports:
-      - "32777:3306"
-    volumes_from:
-      - exam_mysql_data
-    env_file: tds-docker.env
-  exam_mysql_data:
-    image: mysql
-    entrypoint: /bin/bash
   exam:
     image: fwsbac/tds-exam-service
     ports:
       - "80:8080"
     depends_on:
-      - assessment
-      - session
-      - student
-      - config
-      - exam_mysql
-    env_file:
-      - ./tds-docker.env
+      configuration-service:
+        condition: service_healthy
```

Comment:
The `service_healthy` condition tells docker-compose to hold off on standing up this service until after the configuration-service passes a healthcheck.